src/client/sandboxed-jquery.js

```diff
@@ -1719,7 +1719,11 @@ export function init (window, undefined) {
                     container.style.zoom = 1;
                 }
 
-                body.removeChild(container);
+                // NOTE: in some cases Firefox can't remove container div,
```

Comment:
IMHO the current version of the comment just duplicates the line of code. It doesn't describe why it happens